tests/models/xapi/fields/test_context.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for the xAPI context fields"""
+
+import pytest
+from pydantic import ValidationError
+
+from ralph.models.xapi.constants import XAPI_EXTENSION_AGENT, XAPI_EXTENSION_SESSION
+from ralph.models.xapi.fields.context import ContextExtensionsField
+
+
+def test_models_xapi_fields_context_context_field_with_valid_session():
+    """Tests that a valid context field does not raise ValidationErrors."""
+
+    try:
+        ContextExtensionsField(
+            **{XAPI_EXTENSION_SESSION: "abcdef0123456789abcdef0123456789"}
```

Comment:
Could you also use parametrization even if there is only one parameter?

Reply:
Good Idea) Thanks) updated)